src/Nox.Generator/Infrastructure/Persistence/DbContext.template.cs

```diff
@@ -96,6 +96,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         foreach (var entity in _noxSolution.Domain!.Entities)
         {
             Console.WriteLine($"{{className}} Configure database for Entity {entity.Name}");
+            ConfigureEnumeratedAttributes(modelBuilder, entity);
```

Comment:
Can we add a test for this functionality?

Reply:
These are covered in the CountriesControllerTests, but I've now included lower level tests on Nox Types level.